cloud/cmd/iptablesManager/app/options/options.go

```diff
@@ -0,0 +1,38 @@
+/*
+Copyright 2021 The KubeEdge Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+   http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package options
+
+import (
+	cliflag "k8s.io/component-base/cli/flag"
+)
+
+// IptablesManagerOptions config
+type IptablesManagerOptions struct {
+	KubeConfig string
+}
+
+// NewIptablesManagerOptions returns options object
+func NewIptablesManagerOptions() *IptablesManagerOptions {
+	return &IptablesManagerOptions{}
```

Comment:
We need to add a default value to the `KubeConfig`

Reply:
I think no need to add here since the containered deployments can use inClusterConfig to build clients. If we add a default value here, maybe we also need to add volumes to mount the kubeconfig like "/root/.kube/config". 

See https://github.com/kubeedge/kubeedge/blob/7ad5dda0ff492dbf19da24252d54cdfc7ddfbc11/vendor/k8s.io/client-go/tools/clientcmd/client_config.go#L616